src/providers/whatsapp/whatsapp.service.ts

```diff
@@ -37,96 +37,147 @@ export class WhatsappService {
       });
   }
 
-  onWaitQrCode: CatchQR = (qrCode) => {
-    this.socketGateway.broadcast(EventTypes.QR_CODE, qrCode);
+  private onWaitQrCode: CatchQR = (base64Qr) => {
+    this.socketGateway.broadcast(EventTypes.QR_CODE, base64Qr);
   };
 
+  async showStatus() {
+    const response = Promise.all([
+      (await this.client.getHostDevice()).wid._serialized,
+      await this.client.getBatteryLevel(),
+      await this.client.isConnected(),
+      await this.client.isLoggedIn(),
+      await this.client.getWAVersion(),
```

Comment:
usa o await ao inves do then 
```suggestion
    const response = await Promise.all([
      (await this.client.getHostDevice()).wid._serialized,
      await this.client.getBatteryLevel(),
      await this.client.isConnected(),
      await this.client.isLoggedIn(),
      await this.client.getWAVersion(),
```

Reply:
na verdade da um problema pq essa primeira função precisa desses parenteses e sem o await o eslint não deixa ficar os parenteses

Reply:
resolvi de um jeito